src/main/java/com/google/cloud/tools/managedcloudsdk/ManagedCloudSdk.java

```diff
@@ -177,14 +185,48 @@ public SdkUpdater newUpdater() {
 
   /** Get a new {@link ManagedCloudSdk} instance for @{link Version} specified. */
   public static ManagedCloudSdk newManagedSdk(Version version) throws UnsupportedOsException {
+    OsInfo osInfo = OsInfo.getSystemOsInfo();
     return new ManagedCloudSdk(
         version,
-        Paths.get(System.getProperty("user.home"), ".google-cloud-tools-java", "managed-cloud-sdk"),
-        OsInfo.getSystemOsInfo());
+        getOsSpecificManagedSdkHome(osInfo.name(), System.getProperties(), System.getenv()),
+        osInfo);
   }
 
   /** Convenience method to obtain a new LATEST {@link ManagedCloudSdk} instance. */
   public static ManagedCloudSdk newManagedSdk() throws UnsupportedOsException {
     return newManagedSdk(Version.LATEST);
   }
+
+  @VisibleForTesting
+  static Path getOsSpecificManagedSdkHome(
+      OsInfo.Name osName, Properties systemProperties, Map<String, String> environment) {
+    Path userHome = Paths.get(systemProperties.getProperty("user.home"));
+    Path cloudSdkPartialPath = Paths.get("google-cloud-tools-java", "managed-cloud-sdk");
+    Path xdgPath = userHome.resolve(".cache").resolve(cloudSdkPartialPath);
+
+    switch (osName) {
+      case WINDOWS:
+        String localAppData = environment.get("LOCALAPPDATA");
+        if (localAppData == null || localAppData.trim().isEmpty()) {
+          logger.warning("LOCALAPPDATA environment is invalid or missing");
+          return xdgPath;
+        }
+        return Paths.get(localAppData).resolve(cloudSdkPartialPath);
```

Comment:
Is it worth doing an exists-check on the LOCALAPPDATA path, similar to the MacOS case?

Reply:
yeah probably.